compiler/rustc_codegen_cranelift/Cargo.toml

```diff
@@ -15,8 +15,8 @@ cranelift-native = { version = "0.98" }
 cranelift-jit = { version = "0.98", optional = true }
 cranelift-object = { version = "0.98" }
 target-lexicon = "0.12.0"
-gimli = { version = "0.27.2", default-features = false, features = ["write"]}
-object = { version = "0.30.3", default-features = false, features = ["std", "read_core", "write", "archive", "coff", "elf", "macho", "pe"] }
+gimli = { version = "0.28.0", default-features = false, features = ["write"]}
+object = { version = "0.32.0", default-features = false, features = ["std", "read_core", "write", "archive", "coff", "elf", "macho", "pe"] }
```

Comment:
These need to stay in sync with cranelift, so updating them is not possible right now.

Reply:
Thanks, I will reverse it